### PR TITLE
updated contact form

### PR DESCRIPTION
**Pull request by himanshujaidka**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -130,25 +130,66 @@ <h3>Open Source Programs we are a part of </h3>
                         <div class="inside-contact">
                             <h2>Get In <span>Touch</span></h2>
                             <h3><span id="confirm"></h3>
-                            <p>Name</p>
+                            <p>First Name *</p>
                             <input id="txt_name" type="text" class="contact-input" required>
-                            <p>Subject</p>
+                            <p>LAst Name *</p>
```
Comment (smaranjitghose) [inline index.html:135]: Please be cautious of spelling errors

File: index.html [hunk 2]
```diff
@@ -130,25 +130,66 @@ <h3>Open Source Programs we are a part of </h3>
...
+                                    </label>
+                                  </div>
+                                  <div class="item">
+                                    <label for="4">
+                                      <input class="radio" type="radio" name="feedback" id="4" value="4">
+                                      <span>😍</span>
+                                    </label>
+                                  <div>
+                              </div>
+                            </label><br><br>
```
Comment (smaranjitghose) [inline index.html:192]: Please ensure that you indent code blocks properly. In the long term, it messes up the readability of code

File: index.html [hunk 3]
```diff
@@ -130,25 +130,66 @@ <h3>Open Source Programs we are a part of </h3>
                         <div class="inside-contact">
                             <h2>Get In <span>Touch</span></h2>
                             <h3><span id="confirm"></h3>
-                            <p>Name</p>
+                            <p>First Name *</p>
```
Comment (smaranjitghose) [inline index.html:133]: Please do not put the asterisk `*`. I mentioned this in the last PR as well

File: assets/css/style.css [hunk 4]
```diff
@@ -975,4 +975,29 @@ footer {
...
\ No newline at end of file
+}
+
+/*emoji feedback*/
+.app {
+  width: 90%;
+  width: 450px;
+  
+}
+  .container-emoji {
```
Comment (smaranjitghose) [inline assets/css/style.css:986]: Comment on each section more!

File: assets/css/style.css [hunk 5]
```diff
@@ -975,4 +975,29 @@ footer {
...
+  .radio {
+  display: none;
+  }
+  .radio ~ span {
+  font-size: 3rem;
+  filter: grayscale(100);
+  cursor: pointer;
+  transition: 0.3s;
+  }
+
```
Comment (smaranjitghose) [inline assets/css/style.css:998]: Comment more!

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
The phone number Should always be placed above the message field

**Comment by smaranjitghose** [comment]
Selecting the last emoji again displaces the send button. Please fix this. I explained in the last PR that this not preferred in terms of UX

**Comment by smaranjitghose** [comment]
The first emoji should not be used in a professional scenario

**Comment by smaranjitghose** [comment]
Add Helper Text instead of headings

**Comment by smaranjitghose** [comment]
I already told you to remove the Get In Touch Heading from the form and put it above where the section begins

**Comment by smaranjitghose** [comment]
Too much description in Captcha

**Comment by smaranjitghose** [comment]
Please refer [this](https://girlscriptboilerplate.netlify.app/contactus). The code is also open source and on my Github

**Comment by himanshujaidka** [comment]
okay, give me some time

**Comment by smaranjitghose** [comment]
> okay, give me some time

Sure

**Pull request closed without merging**